colossalai/tensor/modules/embedding.py

```diff
@@ -0,0 +1,36 @@
+from .colo_module import ColoModule
+from colossalai.tensor import ComputePattern, distspec
+from colossalai.core import global_context as gpc
+from colossalai.context.parallel_mode import ParallelMode
+
+class ColoEmbedding(ColoModule):
+    def __init__(self):
+        super(ColoEmbedding, self).__init__()
+        self._register_shard_params(['weight'])
```

Comment:
why not register bias? Because its size is small?

Reply:
We just split embedding's lookup table, the weight, when doing TP.